### ISSUE TITLE
some question about paper

### ISSUE THREAD

**Issue by aohenuo** [EvanZhuang/vector-icl#1]
This paper is excellent! However, I have a few questions about it:

First, from the illustration in the paper, it seems that there is a pre-training stage where part of the text is compressed into blocks, followed by a task fine-tuning stage where the block’s encoded information is aligned. My question is: during the task fine-tuning stage, is the entire ICD converted into blocks, or is only part of the information in the ICD converted into blocks?

The paper mentions an image classification task. How is this encoded into blocks? Is it done using visual-text feature joint encoders like CLIP, or is only the text portion encoded?

**Comment by EvanZhuang** [comment]
Thanks for your question! 

During both pretraining and task finetuning, we encode the entire input (regardless of its modality) into a single vector, then align the vector with a projector. So, for example with text summarization task, what you'd have is [Representation]'s summarization is [label text] as the model's input. 

We did not mention an image classification, but suppose you'd want to do that, the same procedure would most likely work as well. You can encode images into representations with any encoder (such as CLIP), then align it with projectors.

**Comment by aohenuo** [comment]
Thank you very much for your answer!
